examples/terms-modal-banner-dyanmo/template.yaml

```diff
@@ -0,0 +1,356 @@
+  MiroBannerTermsAcceptersTable:
+    Type: "AWS::Serverless::SimpleTable"
+    Properties:
+      TableName:
+        !Join [
+          "-",
+          [
+            "miro-terms-banner-accepters-database",
+            !Select [
+              4,
+              !Split ["-", !Select [2, !Split ["/", !Ref AWS::StackId]]],
+            ],
+          ],
+        ]
+      PrimaryKey:
+        Name: id
+        Type: String
+
+  ## IAM Role for the "Read" Lambda Function to query/read the database
+  MiroBannerTermsReadFunctionRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: MiroBannerTermsReadFunctionRole
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action: "sts:AssumeRole"
+      Policies:
+        - PolicyName: MiroBannerTermsDynamoDBReadPolicy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "dynamodb:GetItem"
+                Resource: !GetAtt MiroBannerTermsAcceptersTable.Arn
+
+  ## IAM Role for the "Write" Lambda Function to write on the database
+  MiroBannerTermsWriteFunctionRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: MiroBannerTermsWriteFunctionRole
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action: "sts:AssumeRole"
+      Policies:
+        - PolicyName: MiroBannerTermsDynamoDBWritePolicy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "dynamodb:UpdateItem"
+                Resource: !GetAtt MiroBannerTermsAcceptersTable.Arn
+
+  ## Lambda "Read" Function
+  MiroBannerTermsReadFunction:
+    Type: "AWS::Serverless::Function"
+    Properties:
+      FunctionName:
+        !Join [
+          "-",
+          [
+            "MiroBannerTermsReadFunction",
+            !Select [
+              4,
+              !Split ["-", !Select [2, !Split ["/", !Ref AWS::StackId]]],
+            ],
+          ],
+        ]
+      Description: !Sub 'Lambda function for the "Miro Terms Modal app". It reads data from the DynamoDB table "${MiroBannerTermsAcceptersTable}"'
+      Handler: app.handler
+      Runtime: nodejs16.x
+      CodeUri: readApp/
+      Timeout: 10
+      Role: !GetAtt MiroBannerTermsReadFunctionRole.Arn
+      Environment:
+        Variables:
+          TABLE_NAME: !Ref MiroBannerTermsAcceptersTable
+
+  ## Lambda "Write" Function
+  MiroBannerTermsWriteFunction:
+    Type: "AWS::Serverless::Function"
```

Comment:
true. Leaving this as is since I don't think the effort is worth it. But something we can revisit in the future if we think it's worth the refactor / retest effort.